server/dotnet/Controllers/PaymentsController.cs

```diff
@@ -57,6 +57,7 @@ public async Task<IActionResult> CreateCheckoutSession()
             //  [billing_address_collection] - to display billing address details on the page
             //  [customer] - if you have an existing Stripe Customer ID
             //  [customer_email] - lets you prefill the email input in the form
+            //  [automatic_tax] - to automatically calculate consumer tax in the Checkout page
```

Comment:
Instead of consumer tax we should probably use "sales tax, VAT, and GST" that we use in docs elsewhere